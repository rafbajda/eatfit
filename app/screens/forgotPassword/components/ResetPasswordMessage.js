import React from 'react';
import { Thumbnail } from 'native-base';
import I18n from 'i18n-js';
import {
    RestPasswordMainMessageText,
    ResetPasswordInfoMessageText,
    iconStyle,
    ResetPasswordMessageContainer
} from '../styles/forgotPasswordStyles';

const logoImage = require('../../../assets/images/reset_password.jpeg');

const ResetPasswordMessage = () => {
    const { t } = I18n;
    return (
        <ResetPasswordMessageContainer>
            <Thumbnail square source={logoImage} style={iconStyle} />

            <RestPasswordMainMessageText>{t('info.resetPassword')}</RestPasswordMainMessageText>
            <ResetPasswordInfoMessageText>
                {t('info.resetPasswordDetails')}
            </ResetPasswordInfoMessageText>
        </ResetPasswordMessageContainer>
    );
};

export default ResetPasswordMessage;
