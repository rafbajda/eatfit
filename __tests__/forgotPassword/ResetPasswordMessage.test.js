import 'react-native';
import React from 'react';
import renderer from 'react-test-renderer';
import NavigationTestUtils from 'react-navigation/NavigationTestUtils';
import ResetPasswordMessage from "../../app/screens/forgotPassword/components/ResetPasswordMessage";

describe('ResetPasswordMessage snapshot', () => {
    jest.useFakeTimers();
    beforeEach(() => {
        NavigationTestUtils.resetInternalState();
    });

    it('renders component', async () => {
        const tree = renderer.create(<ResetPasswordMessage />).toJSON();
        expect(tree).toMatchSnapshot();
    });
});
