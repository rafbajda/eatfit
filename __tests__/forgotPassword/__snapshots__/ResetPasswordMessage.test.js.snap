// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ResetPasswordMessage snapshot renders component 1`] = `
<RCTScrollView
  automaticallyAdjustContentInsets={false}
  className="sc-htpNat KtFsv"
  contentContainerStyle={
    Array [
      Object {
        "padding": undefined,
      },
      Object {
        "alignItems": "center",
        "display": "flex",
        "flexDirection": "column",
        "padding": 25,
      },
    ]
  }
  contentInset={
    Object {
      "bottom": 0,
    }
  }
  enableAutomaticScroll={true}
  enableOnAndroid={false}
  enableResetScrollToCoords={true}
  extraHeight={75}
  extraScrollHeight={0}
  getScrollResponder={[Function]}
  handleOnScroll={[Function]}
  keyboardDismissMode="interactive"
  keyboardOpeningTime={250}
  keyboardShouldPersistTaps="handled"
  keyboardSpace={0}
  onScroll={[Function]}
  resetKeyboardSpace={[Function]}
  resetScrollToCoords={
    Object {
      "x": 0,
      "y": 0,
    }
  }
  scrollEventThrottle={1}
  scrollForExtraHeightOnAndroid={[Function]}
  scrollIntoView={[Function]}
  scrollToEnd={[Function]}
  scrollToFocusedInput={[Function]}
  scrollToPosition={[Function]}
  showsVerticalScrollIndicator={true}
  style={
    Object {
      "backgroundColor": "transparent",
      "flex": 1,
    }
  }
  update={[Function]}
  viewIsInsideTabBar={false}
>
  <View>
    <Image
      source={1}
      square={true}
      style={
        Array [
          Object {
            "borderRadius": 0,
            "height": 56,
            "width": 56,
          },
          Object {
            "height": 180,
            "marginBottom": 25,
            "width": 180,
          },
        ]
      }
    />
    <Text
      className="sc-bdVaJa eftihw"
      style={
        Object {
          "color": "#000",
          "fontFamily": "System",
          "fontSize": 16,
        }
      }
      uppercase={false}
    >
      [missing "en.info.resetPassword" translation]
    </Text>
    <Text
      className="sc-bwzfXH bAgBGf"
      style={
        Object {
          "color": "#000",
          "fontFamily": "System",
          "fontSize": 16,
        }
      }
      uppercase={false}
    >
      [missing "en.info.resetPasswordDetails" translation]
    </Text>
  </View>
</RCTScrollView>
`;
